src/olympia/reviewers/views.py

```diff
@@ -725,6 +725,13 @@ def review(request, addon, channel=None):
         flags=flags,
         form=form,
         format_matched_rules=formatted_matched_rules_with_files_and_data,
+        has_versions_with_due_date_in_other_queue=addon.versions.exclude(
```

Comment:
oh, forget to add this comment: see mat's previous review - we need `addon.versions(manager='unfiltered_for_relations')` not just `addon.versions` to include deleted  versions too.